src/pages/sighting/statusCard/PipelineStepProgressGrid.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import Grid from '@material-ui/core/Grid';
+
+import ProgressCard from '../../../components/progress/ProgressCard';
+import QueueCard from '../../../components/progress/QueueCard';
+import TimeRemainingCard from '../../../components/progress/TimeRemainingCard';
+
+const cardStyle = {
+  height: '100%',
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+};
+
+export default function PipelineStepProgressGrid({
+  pipelineStep,
+  style,
+}) {
+  const { ahead, eta, progress } = pipelineStep || {};
+
+  return (
+    <Grid container spacing={1} style={style}>
+      <Grid item xs={6}>
+        <QueueCard ahead={ahead} style={cardStyle} />
```

Comment:
Why no <StartDateCard> in this one?

Reply:
I think this is supposed to say, "Why no StartDateCard in this one?"

Reply:
The start date is displayed as part of the In Progress text for the step, so a separate card would have been redundant.